wallet-new/src/Cardano/Wallet/Kernel/Addresses.hs

```diff
@@ -65,6 +67,11 @@ instance Buildable CreateAddressError where
     build (CreateAddressHdRndAddressSpaceSaturated hdAcc) =
         bprint ("CreateAddressHdRndAddressSpaceSaturated " % F.build) hdAcc
 
+instance Show CreateAddressError where
+    show = formatToString build
+
+instance Exception CreateAddressError
```

Comment:
Side note: we should open a ticket to be sure we track down all the possible exceptions that the `Kernel` (and/or the `WalletLayer`) can throw, so that we can catch this into the `wai` exception handler at the top level and @KtorZ 's squad will eventually be able to curate them into proper V1 error types.